### ISSUE TITLE
Cannot Uninstall or Update

### ISSUE THREAD

**Issue by sdflyguy** [seydx/homebridge-camera-ui#579]
**Describe the bug**
Initially, I had the error:

FFmpeg prebuffer process exited with error! (null)

Found a solved issue with that same error. The fix was to reinstall via command: 

sudo npm i -g homebridge-camera-ui@latest --unsafe-perm

After running that command, I now get the error:

[05/06/2022, 14:41:48] ====================
[05/06/2022, 14:41:48] ERROR LOADING PLUGIN homebridge-camera-ui:
[05/06/2022, 14:41:48] Error: Cannot find package '/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/camera.ui/' imported from /usr/local/lib/node_modules/homebridge-camera-ui/src/platform.js
    at new NodeError (node:internal/errors:372:5)
    at legacyMainResolve (node:internal/modules/esm/resolve:341:9)
    at packageResolve (node:internal/modules/esm/resolve:941:14)
    at moduleResolve (node:internal/modules/esm/resolve:1003:20)
    at defaultResolve (node:internal/modules/esm/resolve:1218:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36)
[05/06/2022, 14:41:48] ====================

**To Reproduce**
Any attempt to reinstall, upgrade, or uninstall reproduces this error. 

**Expected behavior**

**Logs**
DIR: /usr/local
CMD: sudo -E -n npm uninstall -g homebridge-camera-ui

npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm ERR! code ENOTEMPTY
npm ERR! syscall rename
npm ERR! path /usr/local/lib/node_modules/homebridge-camera-ui
npm ERR! dest /usr/local/lib/node_modules/.homebridge-camera-ui-qDB49Qw3
npm ERR! errno -39
npm ERR! ENOTEMPTY: directory not empty, rename '/usr/local/lib/node_modules/homebridge-camera-ui' -> '/usr/local/lib/node_modules/.homebridge-camera-ui-qDB49Qw3'

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2022-06-05T19_47_38_478Z-debug-0.log

Operation failed. Please review log for details.

**Screenshots**

**Environment**
- Node.js Version: v16.15.1
- Homebridge Version: v1.4.1
- Camera UI Version: v5.0.27
- Operating System: pi
- Browser: Safari

**Comment by seydx** [comment]
>Found a solved issue with that same error. The fix was to reinstall via command:

>sudo npm i -g homebridge-camera-ui@latest --unsafe-perm

**Comment by sdflyguy** [comment]
Not sure why you closed this issue. But I did figure out by renaming a directory, I could update via sudo npm i -g homebridge-camera-ui@latest --unsafe-perm.

However, I still get the same error after multiple updates and re-installs..

[11/06/2022, 22:24:18] [CameraUI]  ERROR  Driveway: FFmpeg prebuffer process exited with error! (null) - [tcp @ 0x31be460] Connection to tcp://192.168.1.156:554?timeout=0 failed: Connection refused - rtsp://jjburros:L9ugo32dsr@192.168.1.156/live: Connection refused -


Here is my config..

{
            "name": "CameraUI",
            "port": 8081,
            "logLevel": "info",
            "mqtt": {
                "active": false,
                "tls": false,
                "port": 1883
            },
            "http": {
                "active": false,
                "port": 7272,
                "localhttp": false
            },
            "smtp": {
                "active": false,
                "port": 2727,
                "space_replace": "+"
            },
            "ftp": {
                "active": false,
                "useFile": false,
                "port": 5050
            },
            "ssl": {
                "active": false
            },
            "cameras": [
                {
                    "name": "Driveway",
                    "manufacturer": "Wyze",
                    "model": "v3",
                    "motion": true,
                    "unbridge": true,
                    "hsv": true,
                    "prebuffering": true,
                    "prebufferLength": 4,
                    "hksvConfig": {
                        "source": "-i rtsp://***:***@192.168.**/live",
                        "audio": true
                    },
                    "videoConfig": {
                        "source": "-i rtsp://***:***@192.168.**/live",
                        "subSource": "-i rtsp://***:***@192.168.**/live",
                        "stillImageSource": "-i rtsp://***:***@192.168.1**/live",
                        "rtspTransport": "tcp",
                        "maxStreams": 3,
                        "vcodec": "copy",
                        "acodec": "libfdk_aac",
                        "audio": true
                    },
                    "videoanalysis": {
                        "active": true
                    },
                    "motionTimeout": 15,
                    "smtp": {
                        "email": "Driveway"
                    },
                    "mqtt": {}
                }
            ],
            "platform": "CameraUI",
            "atHomeSwitch": false,
            "options": {
                "videoProcessor": "ffmpeg"
            }

**Comment by seydx** [comment]
> Not sure why you closed this issue. But I did figure out by renaming a directory, I could update via sudo npm i -g homebridge-camera-ui@latest --unsafe-perm.
> 
> However, I still get the same error after multiple updates and re-installs..
> 
> [11/06/2022, 22:24:18] [CameraUI] ERROR Driveway: FFmpeg prebuffer process exited with error! (null) - [tcp @ 0x31be460] Connection to tcp://192.168.1.156:554?timeout=0 failed: Connection refused - rtsp://jjburros:L9ugo32dsr@192.168.1.156/live: Connection refused -
> 
> Here is my config..
> 
> { "name": "CameraUI", "port": 8081, "logLevel": "info", "mqtt": { "active": false, "tls": false, "port": 1883 }, "http": { "active": false, "port": 7272, "localhttp": false }, "smtp": { "active": false, "port": 2727, "space_replace": "+" }, "ftp": { "active": false, "useFile": false, "port": 5050 }, "ssl": { "active": false }, "cameras": [ { "name": "Driveway", "manufacturer": "Wyze", "model": "v3", "motion": true, "unbridge": true, "hsv": true, "prebuffering": true, "prebufferLength": 4, "hksvConfig": { "source": "-i rtsp://_**:**_@192.168.**/live", "audio": true }, "videoConfig": { "source": "-i rtsp://**_:_**@192.168.**/live", "subSource": "-i rtsp://_**:**_@192.168.**/live", "stillImageSource": "-i rtsp://**_:_**@192.168.1**/live", "rtspTransport": "tcp", "maxStreams": 3, "vcodec": "copy", "acodec": "libfdk_aac", "audio": true }, "videoanalysis": { "active": true }, "motionTimeout": 15, "smtp": { "email": "Driveway" }, "mqtt": {} } ], "platform": "CameraUI", "atHomeSwitch": false, "options": { "videoProcessor": "ffmpeg" }

Because its not the same issue 

ffmpeg throws a connection refused error, so maybe the camera source is not valid or authentication failed

can you check it with eg VLC?

**Comment by sdflyguy** [comment]
VLC pulls the stream just fine. I altered the passwords to not include capital letters. That allowed the feed to pull into the Cam UI Interface. But I am still having problems with pre buffering and video analysis errors through home bridge.

My coding experience is low. Lets start from the top. I have the Homebridge Camera FFmpeg plugin installed. Is that all I need along with the Camera Ui plugin? Everything was working great until a few months ago when I believe something changed with FFmpeg. I've reinstalled everything and set it up as I had before. I'm using a RTSP feed for 3 Wyze v3 Cams.
